### DISCUSSION TITLE
STARs missing waypoints -> Am I doing something wrong or is this normal?

### DISCUSSION THREAD

**Discussion by philhzss** [flybywiresim/aircraft#4861] [Help] [answered]
Hi!

Just a quick question. I've looked at a bunch of tutorials and videos about this mod so I think I'm doing things mostly right, but it could very well be my fault!!

Flew KLGA-CYUL last night. I loaded in the CARTER4 arrival for 24R. No transitions were available but as far as I know there aren't any in real life either on this route. The STAR loaded but I noticed that after DUNUP (the point where the STAR splits left or right), I had nothing until the runway IAF. So MATEP ELTAT SILDO etc were all missing.

![image](https://user-images.githubusercontent.com/41757256/119032972-b6dcbb80-b97a-11eb-98e4-a833987021a8.png)

I thought this might be just a Navigraph data / MSFS data bug so I tried in a bunch of other planes... and the results are odd. In the working title CJ4, no issue, it all loaded properly. In the DA62 (using the working title G1000), all loaded properly too. However the default 747 had the same glitch as the FBW A32X.

Did I install my navdata wrong, or is this a known issue?

Thanks a lot!

**Comment by ExampleWasTaken** [comment, marked as answer]
The A32NX still uses the default flight plan manager, which has a lot of bugs. As far as I know the WT CJ4 and the WT G1000, both use a custom flight plan manager.
A custom flight plan manager for the A32NX is currently in development and will be the next version to come to experimental.

**Reply by philhzss** [reply]
Okay thank you very much! I'll keep an eye on the release notes and might try the experimental build one day. Thank you!

**Reply by ExampleWasTaken** [reply]
> 
> 
> Okay thank you very much! I'll keep an eye on the release notes and might try the experimental build one day. Thank you!

I recommend joining the official Discord server [here](https://discord.gg/flybywire) to get the latest news, including when the experimental version is activated again.
